atd-etl/cris_import/cris_import.py

```diff
@@ -242,6 +316,59 @@ def download_extract_archives():
         return zip_tmpdir
 
 
+def specify_extract_location():
+    zip_files = glob.glob('/app/development_extracts/*.zip')
+    if not zip_files:
+        return False
+
+    zip_tmpdir = tempfile.mkdtemp()
+
+    for file_to_copy in zip_files:
+        shutil.copy(file_to_copy, zip_tmpdir)
+
+    return zip_tmpdir
+
+
+
```

Comment:
my linter is saying there are two many lines between some of these functions

Reply:
Great catch. I've linted the entire application with `black` now in 39cf8557641193901f79c75bf6bd34df3528e288.